### DISCUSSION TITLE
Support for Austrian and European Vat Requirements -- Feature Request

### DISCUSSION THREAD

**Discussion by GVLLIFESTYLE** [baimard/gestion#234] [Ideas]
Austria:

Date of issuance
A unique, sequential number
VAT number of the supplier
VAT number of the customer (if supply exceeds EUR10,000)
Full name and address of the supplier and customer
Full description of the goods or services provided
Details of quantities of goods, if applicable
A date of the supply if different from the invoice date
The net, taxable value of the supply
The VAT rate applied, and the amount of VAT (the amount of VAT must be stated in EUR)
Details to support zero VAT – export, reverse charge or intra-community supply
Details of any margin schemes applied e.g. works of art, collector’s items and antiques.
The total, gross value of the invoice

EU:

The date of the invoice
A unique invoice number, which should be sequential
The name and address of the supplier and the customer
The VAT number of the supplier
A description of the goods supplied, including quantities
The date of the supply of the goods or services where materially different from the date of the invoice
The net amount charged
The [EU VAT rate](https://www.avalara.com/vatlive/en/eu-vat-rules/distance-selling/fulfilment-amazon-eu-vat/european-union-eu-vat-rates.html) charged, and the amount, and the gross charge
Foreign currency rate if an alternative currency to the national currency of the supplier is used (this rate must be based on the local tax law e.g. a Central Bank rate in France

**Comment by baimard** [comment]
What's your issue ?

**Comment by GVLLIFESTYLE** [comment]
IF that is already all supported, I probably haven't yet firgured all out.
In this case, please close the feature request.

**Comment by baimard** [comment]
I don't This a list of what you don't see in the application ? could you explain more please ?

**Comment by GVLLIFESTYLE** [comment]
Well I was not using the app alot yet.
I just thought I add it as a request in case it misses something.
Lets close that issue.
I will reopen another one if I cannot somehow handle the list items.

**Comment by baimard** [comment]
No no it's very good information, I just change it in conversation to discuss maybe about some additional features.

Do you agree ?

**Comment by GVLLIFESTYLE** [comment]
OK :)
